Sources/SUIComponents/Organisms/EditableListView.swift

```diff
@@ -0,0 +1,104 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+extension Components.Organisms {
+    public struct EditableListView2<Content: View>: View {
+        let content: Content
+
+        public init(@ViewBuilder _ content: () -> Content) {
+            self.content = content()
+        }
+
+        public var body: some View {
+            VStack {
+                content
+            }
+        }
+    }
+    public struct EditableListView: View {
+
+        fileprivate static var isVertical: Bool { FontMetrics.scaler > 1.36 }
+
+        public struct Model {
+            public let title: String
+
+            public let rowButtonTitle: String
+            public let startEditingButtonTitle: String
+            public let startEditingButtonImage: Image
+            public let stopEditingButtonTitle: String
+            public let stopEditingButtonImage: Image
+
+            public let views: [any View]
+            public let didTapRow: (Int) -> Void
+
+            public init(title: String, rowButtonTitle: String, startEditingButtonTitle: String, startEditingButtonImage: Image, stopEditingButtonTitle: String, stopEditingButtonImage: Image, views: [any View], didTapRow: @escaping (Int) -> Void) {
+                self.title = title
+                self.rowButtonTitle = rowButtonTitle
+                self.startEditingButtonTitle = startEditingButtonTitle
+                self.startEditingButtonImage = startEditingButtonImage
+                self.stopEditingButtonTitle = stopEditingButtonTitle
+                self.stopEditingButtonImage = stopEditingButtonImage
+                self.views = views
+                self.didTapRow = didTapRow
+            }
+        }
+
+        @State var isEditing: Bool
+
+        let model: Model
+
+        public init(model: Model, isEditing: Bool = false) {
+            self.model = model
+            self.isEditing = isEditing
+        }
+
+        public var body: some View {
+            VStack(alignment: .leading, spacing: 0) {
+                Text(model.title, style: .H5)
+                    .padding(.spacer16)
+                ForEach(model.views.indices, id: \.self) { index in
+                    Components.Atoms.Stack(
+                        axis: EditableListView.isVertical ? .vertical(alignment: .leading) : .horizontal(alignment: .center),
+                        spacing: 0) {
+                            model.views[index]
+                                .typeErased
+                                .frame(minHeight: 21) // Minimum height of an Edit button
```

Comment:
Could this '21'  be a library constant or a property instead?

Reply:
Updated in [9369659](https://github.com/hmrc/ios-swiftui-components/pull/33/commits/9369659d6dd224858f18586187308e9890226961)